scripts/debug_tools/prepare_all_cmd_for_ctu.py

```diff
@@ -17,24 +20,38 @@
 import prepare_compiler_info
 import prepare_analyzer_cmd
 
+LOG = logging.getLogger(__name__)
 
-def execute(cmd):
-    print("Executing command: " + ' '.join(cmd))
-    try:
-        proc = subprocess.Popen(cmd,
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.PIPE)
-        out, err = proc.communicate()
+handler = logging.StreamHandler()
+formatter = logging.Formatter('[%(levelname)s %(asctime)s] - %(message)s',
+                              '%Y-%m-%d %H:%M')
+handler.setFormatter(formatter)
 
-        print("stdout:\n\n" + out.decode("utf-8"))
-        print("stderr:\n\n" + err.decode("utf-8"))
+LOG.setLevel(logging.INFO)
+LOG.addHandler(handler)
 
+
+def execute(cmd, cwd=None, env=None):
```

Comment:
Having `cwd` and `env` here is useful. The output format change is not necessary though, just made my output more consistent with CodeChecker's output. Are we relying on the previous format somewhere? Should I revert the changes?

Reply:
It is okay like this, we don't depend on the output anywhere.